lib/ex_doc/formatter/html/autolink.ex

```diff
@@ -617,6 +617,25 @@ defmodule ExDoc.Formatter.HTML.Autolink do
     end
   end
 
+  defp hex_lib_dirs() do
+    [{hex_ebin_path(), @elixir_docs <> "hex/"}]
+  end
+
+  defp hex_ebin_path() do
+    # TODO: Private Elixir API
+    # TODO: Not available when running ex_doc as escript
```

Comment:
I'm not sure how to best solve it, is there a way to detect at compile time that we're building an escript? If there is I could:

```elixir
if compiling_escript?() do
  defp hex_lib_dirs(), do: []
else
  defp hex_lib_dirs(), do: [{hex_ebin_path(), @elixir_docs <> "hex/"}]
end
```